pkgs/tools/typesetting/tex/texlive/make-bin-containers.sh

```diff
@@ -0,0 +1,62 @@
+# load realpath
+loadables="$(command -v bash)"
+loadables="${loadables%/bin/bash}/lib/bash"
+enable -f "$loadables/realpath" realpath
+mkdir -p "$out/bin"
+
+# find interpreters
+export interpPerl="$(command -v perl)"
+export interpJava="$(command -v java || :)"
+export interpWish="$(command -v wish || :)"
```

Comment:
```suggestion
export interpPerl="$(PATH="$HOST_PATH" command -v perl)"
export interpJava="$(PATH="$HOST_PATH" command -v java || :)"
export interpWish="$(PATH="$HOST_PATH" command -v wish || :)"
```

When using `strictDeps`  or when cross-compiling, the `buildInputs` populate `HOST_PATH` and the `nativeBuildInputs` populate `PATH`. 

(yes, I'm still dreaming of having some cross support  for texlive, and the upcoming format containers prepared in this PR should get us pretty close. The format containers that cannot be  dumpshared (for example because they contain machine-specific lua bytecode for example) could be excluded when combining and generated on first use)

Reply:
Thanks! I always get the paths wrong, and I even consulted `patchSheBangs` at some point. Very easy fix, luckily.